pyrokinetics/local_geometry/LocalGeometry.py

```diff
@@ -113,30 +113,26 @@ def from_global_eq(
         """
         Loads LocalGeometry object from an Equilibrium Object
         """
-        # TODO Currently stripping units from Equilibrium/FluxSurface.
+        # TODO Currently stripping units from Equilibrium/FluxSurface. These should be
+        # added in a later update.
+        # TODO FluxSurface is COCOS 11, this uses something else. Here we switch from
+        # a clockwise theta grid to a counter-clockwise one, and divide any psi
+        # quantities by 2 pi
```

Comment:
Why did you change the units of `psi` here? Do we need `Wb/rad` later on?

Reply:
Ah is this to put it in a ITER standard cocos of 11?

Reply:
Yeah, it's just a COCOS 11 thing. I try to make sure that we convert to COCOS 1/2 at the point we switch to `LocalGeometry`, so hopefully it won't break anything for now. If we're aiming to use IMAS as the standard we'll have to push similar changes there too at some point.